Stratis.Bitcoin/Features/Wallet/WalletManager.cs

```diff
@@ -46,7 +46,7 @@ public class WalletManager : IWalletManager
         // every time we find a trx that credits we need to add it to this lookup
         // private Dictionary<OutPoint, TransactionData> outpointLookup;
 
-        internal Dictionary<Script, HdAddress> keysLookup;
+        public Dictionary<Script, HdAddress> keysLookup;
```

Comment:
You can use the `[assembly: InternalsVisibleTo("Stratis.Bitcoin.Tests")]` attribute instead of making it public
See how AssemblyInfo in Stratis.Bitcoin

Reply:
Added ` [assembly: InternalsVisibleTo("Stratis.Bitcoin.Tests")]'